app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
```

Comment:
- As described in the previous review. The `foreign_key` attribute here is not adding any value to the association since the model already knows from the association that a field will be created with the model in a relationship called `model_id`. Since the `Comment` model knows the `User` model as an `author`, it'll automatically create an FK called `author_id`.

Reply:
Ah thanks, I didn't get it the first time :pray: